reset-migration.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+REPO=$1
+
+# caution make sure anything you want to keep is managed in code
```

Comment:
Maybe add a prompt on this script? ^^

Reply:
tbh it's only really useful if you have very few issues, could just delete it. you get rate limited fairly quickly and it's slow.

when testing it's better to just delete and re-create the repo